src/main/java/com/aws/greengrass/lifecyclemanager/Kernel.java

```diff
@@ -96,6 +98,7 @@
     private static final String DEPLOYMENT_STAGE_LOG_KEY = "stage";
     protected static final ObjectMapper CONFIG_YAML_WRITER =
             YAMLMapper.builder().disable(JsonGenerator.Feature.AUTO_CLOSE_TARGET).build();
+    private static final List<String> supportedCapabilities = Arrays.asList("LARGE_CONFIGURATION");
```

Comment:
can this be moved somewhere else? Deployment service maybe?

Reply:
Supported capabilities are associated with a kernel version, thought kernel will be the first place someone will look for it. Deployment service class is also large (approx 900 lines right now).

Reply:
A Enum is probably required for this

Reply:
This needs to be empty for now, right? We don't actually support it yet.

Reply:
Correct!!